UnitsNet.Tests/CustomCode/MagneticFieldTests.cs

```diff
@@ -32,5 +32,6 @@ public class MagneticFieldTests : MagneticFieldTestsBase
         protected override double NanoteslasInOneTesla => 1e9;
         protected override double MicroteslasInOneTesla => 1e6;
         protected override double MilliteslasInOneTesla => 1e3;
+        protected override double GaussesInOneTesla => 1e4;
```

Comment:
Verified
https://en.wikipedia.org/wiki/Gauss_(unit)#:~:text=Unit%20conversions,-The%20gauss%20is&text=One%20tesla%20(T)%20corresponds%20to,in%20the%20CGS%2DGaussian%20system.